CHANGELOG.md

```diff
@@ -1,14 +1,19 @@
-# Change Log
+# Changelog
```

Comment:
Thoughts on adding the changelog to the website? (can always be done in another PR)

Reply:
Good point! I'll do separately as I have other changes to docs anyway.